### ISSUE TITLE
Need a Documentation Website using Docsaurus

### ISSUE THREAD

**Issue by DARK-art108** [CoinShift-Imaging/CoinShift-Imaging-Box#18]
* We need a website which is particularly a Doc website similar to which we have in other orgs like Red Hat and others.
* https://docusaurus.io/

**Comment by DARK-art108** [comment]
Yeah @shikharcodes we are only having three pages a landing page with logo and a tag line, the above nav bar should have only the doc tab, that's it.

**Comment by DARK-art108** [comment]
@shikharcodes check out https://www.sphinx-doc.org/en/master/ it is much easier then docsaures.

**Comment by PragatiBhat** [comment]
I am gssoc21 participant i want to work on this issue please let me know what needs to be done
